### PR TITLE
Verify repomd hashes with a task

### PR DESCRIPTION
**Pull request by poncovka**
Call the `VerifyRepomdHashesTask` task to verify repomd hashes.

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Thank you, looks good to me.

File: pyanaconda/modules/payloads/payload/dnf/validation.py [hunk 1]
```diff
@@ -29,7 +30,38 @@
...
+    def run(self):
+        """Run the task.
+
+        :return: a validation report
+        """
+        report = ValidationReport()
+
+        if not self._dnf_manager.verify_repomd_hashes():
+            report.error_messages.append(_(
+                "Some of the repomd.xml files have changed or are unreachable."
```
[the remark this answers is not in the document]
Comment (poncovka) [reply pyanaconda/modules/payloads/payload/dnf/validation.py:58]: The message won't be visible in GUI or TUI, so I will keep it as it is now.

**Comment by poncovka** [comment]
Rebased.

**Pull request merged**